bigtop_toolchain/manifests/isal.pp

```diff
@@ -0,0 +1,65 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more

```

Comment:
The line endings looks CRLF in this file. It should be aligned to LF.

Reply:
Okay, I have modified this section to LF